_posts/multiworld/2024-03-10-multiworld-with-nintendont.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: post
+title: "Connecting Randovania to a Real Wii or Wii U with Nintendont Multiworld"
+tags: multiworld prime1 prime2
```

Comment:
this isn't really a multiworld guide

Reply:
I was wondering how to describe it generally - would "Connecting Randovanian to a Real Wii or Wii U" work?

Reply:
yeah that should be fine ~~as long as you spell randovania correctly~~